### PR TITLE
Steve two var describe

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgDescribeTwoVarGraph.vb [hunk 1]
```diff
@@ -87,33 +112,7 @@ Public Class dlgDescribeTwoVarGraph
...
-        ucrSaveGraph.SetSaveTypeAsGraph()
-        ucrSaveGraph.SetDataFrameSelector(ucrSelectorTwoVarGraph.ucrAvailableDataFrames)
-        ucrSaveGraph.SetCheckBoxText("Save Graph")
-        ucrSaveGraph.SetIsComboBox()
-        ucrSaveGraph.SetAssignToIfUncheckedValue("last_graph")
-
-    End Sub
-
-    Private Sub SetRCodeForControls(bReset As Boolean)
+    Public Sub SetRCodeForControls(bReset As Boolean)
```
Comment (lilyclements) [inline instat/dlgDescribeTwoVarGraph.vb]: SetRCode should be Public as it will be called in the sub

**Comment by lilyclements** [comment]
@stevenndungu I made minor layout edits, this is ready to merge

File: instat/dlgDescribeTwoVarGraph.vb (hunk 1 above)
Comment (stevenndungu) [reply instat/dlgDescribeTwoVarGraph.vb]: Really? it is only set for the main dialog and so need not be public. The sub-dialogue has its own setrcode. So it should still be private according to my understanding

File: instat/dlgDescribeTwoVarGraph.Designer.vb [hunk 2]
```diff
@@ -34,9 +34,9 @@ Partial Class dlgDescribeTwoVarGraph
         '
         'cmdOptions
         '
-        Me.cmdOptions.Location = New System.Drawing.Point(10, 197)
+        Me.cmdOptions.Location = New System.Drawing.Point(311, 228)
         Me.cmdOptions.Name = "cmdOptions"
```
Comment (stevenndungu) [inline instat/dlgDescribeTwoVarGraph.Designer.vb]: I changed this to position 10,97 intentionally, based on the the discussion that led to the frequencies dialogues. Where the options buttons were moved to the left just as i did here. For consistency with most of the other dialogues as well.

Comment (lilyclements) [reply instat/dlgDescribeTwoVarGraph.Designer.vb]: Yeah, this is a good point. I don't know where we want it. It gets put in both places in different dialogs, I just want consistency and I can't decide which place is better.
I feel it gets a little "lost" in the place it was in, however, most plot dialogs do have it there!
You can change it back if you want, sorry!

Comment (lilyclements) [reply instat/dlgDescribeTwoVarGraph.Designer.vb]: Thanks for looking at the changes too!

File: instat/dlgDescribeTwoVarGraph.vb (hunk 1 above)
Comment (lilyclements) [reply instat/dlgDescribeTwoVarGraph.vb]: Really? I didn't know that! I'll change it back now!

**Comment by lilyclements** [comment]
@stevenndungu I've made the changes: Changed back to Private and moved the button back

**Review by lilyclements — COMMENTED** [review]
Out of interest, do we need to do the "= New R Function" for every function? Or just for the functions in the SetDefaults() dialog?

I thought the latter, but may be wrong!

**Comment by stevenndungu** [comment]
Setting "New R Function" for every function is necessary since, it creates new instance of the clsRFunction class. Having all set at default makes sure that the parameters of the respective function are cleared and other things... check on `clsRFunction`  class.

File: instat/dlgDescribeTwoVarGraph.Designer.vb (hunk 2 above)
Comment (stevenndungu) [reply instat/dlgDescribeTwoVarGraph.Designer.vb]: Great and thanks as well, I just wanted to check and see the changes made.

**Comment by stevenndungu** [comment]
@lilyclements  this is causing two conflicts..It has changes on summary tables

**Pull request merged**